### ISSUE TITLE
vic executable usage message refers to compile-time settings in vicNl_def.h

### ISSUE THREAD

**Issue by tbohn** [UW-Hydro/VIC#332]
I think the compile-time settings are gone, right?  If so, the entire -o option can go away.  Otherwise, if we want to keep this option, the message just needs to be changed to refer to the correct .h file.

**Comment by jhamman** [comment]
They are not all gone.  When I call the `-o` option, I don't see any reference to `vicNl_def.h`.

```
$ ./vic_classic.exe -o
  VIC Version : 5.0.beta
  VIC Driver  : Classic

  Variable Infiltration Capacity (VIC) macroscale hydrologic
  model version 5.0.beta, Copyright (C) 2014 Land Surface
  HydrologyGroup, Dept. of Civil and Environmental Engineering,
  University of Washington.  VIC comes with ABSOLUTELY NO
  WARRANTY. This is free software, you may redistribute it
  under certain conditions; see LICENSE.txt for details.

  Report Bugs and Issues to : https://github.com/UW-Hydro/VIC/issues
  VIC Users Email Listserve : vic_users@u.washington.edu 


Current Model Settings

COMPILE-TIME OPTIONS (set in .h files)
----------------------------------------

VIC_DRIVER:		Classic

LOG_LEVEL:		5

Maximum Array Sizes:
MAX_BANDS		10
MAX_FRONTS		 3
MAX_FROST_AREAS		10
MAX_LAKE_NODES		20
MAX_ZWTVMOIST		11
MAX_LAYERS		 3
MAX_NODES		50
MAX_VEG			12

MINSOILDEPTH		0.001000
MIN_VEGCOVER		0.000100

MIN_SUBDAILY_STEPS_PER_DAY 4
MAX_SUBDAILY_STEPS_PER_DAY 1440
```

**Comment by tbohn** [comment]
I'm talking about the usage message.  When you type vic_classic.exe without arguments:
Usage: vic_classic.exe [-v | -o | -g <global_parameter_file>]
  v: display version information
  o: display compile-time options settings (set in vicNl_def.h)
  g: read model parameters from <global_parameter_file>.
       <global_parameter_file> is a file that contains all needed model
       parameters as well as model option flags, and the names and
       locations of all other files.

**Comment by jhamman** [comment]
Oh, I see.  Yes we should just change "set in vicNl_def.h" to "set in .h files".

Also, do you know that you can add triple ticks (\`\`\`) to your markdown code so you get syntax highlighting?  It makes these Github posts alot easier to read.  I fixed your last post.

**Comment by tbohn** [comment]
OK, thanks - still learning markdown!  I just copied/pasted that blob into the window.

Meanwhile, I asked whether compile-time options are gone because the values of MAX_* don't really seem like "options" in the sense that they used to be.  Previously, entire modes of operation that changed model physics were set by these "#define" statements, and there wasn't an easy way for a user to know how they were set in the executable that they were using (while the user could check the source code, I actually saw cases where the executable was taken out of the original source code context or the source code had been edited without recompiling - this complicated things!). Those are what are now gone.

But I can see how a user might want to see the values of the MAX* etc are, so I suppose it still serves some purpose to have the -o option.

Anyway, I was just bringing it up to make sure it hadn't been overlooked.

**Comment by jhamman** [comment]
Yeah, no problem.  I think we leave it for now since it isn't breaking anything.  With the addition of extension modules (e.g. routing), which will be added to the executable at run time, we'll want to add those to this info too.

I'll fix the header file line in a PR here shortly.

**Comment by jhamman** [comment]
closed via #333
